components/d2l-hello-world.js

```diff
@@ -0,0 +1,28 @@
+import { LitElement, html } from 'lit-element';
+
+export class D2LHelloWorld extends LitElement {
+
+	static get properties() {
+		return {
+			message: { type: String }
+		};
+	}
+
+	constructor() {
+		super();
+		this.message = 'Hello World from LitElement';
+	}
+
+	render() {
+		return html`
```

Comment:
So I think we did want to add that whitespace fix, since lit doesn't have a special way of handling it?

Reply:
I was looking at the source for the `TemplateResult` and I didn't see any reference to `strip-whitespace`.  I can try it out though.  Stay tuned.

Reply:
Ok - gave it a try and `setAttribute` is not on the returned `TemplateResult` object.  The TemplateResult does not contain DOM per se, but rather a description of what needs to be created.  I think the `html` template function that comes with Polymer must be a bit different than the one that comes from `lit-element`.

Reply:
Ah ok, so we currently don't have a way to solve for that.  Should add that to the list!